package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.js",
   "homepage": "https://github.com/openaustin/",
   "author": "Open Austin",
+  "scripts": {
+    "start": "gulp clean && gulp"
```

Comment:
Do you mean like having npm start run a regular static file server? Should I take the start script out of the package?